### ISSUE TITLE
P1: Parsing Error 

### ISSUE THREAD

**Issue by hgitta** [looker-open-source/look-at-me-sideways#62]
![image](https://user-images.githubusercontent.com/9141252/153502116-49d20a7a-f799-48a8-8687-6daef1c6736b.png)

We have upgraded our looker instance to 22.0.18 and I am getting  `P1: Parsing error: Expected "#", "\"", [ \t\n\r], or [\-+_a-zA-Z0-9.] but "]" found  `in the files that we didn't change in months.

**Comment by fabio-looker** [comment]
Hi @hgitta - I believe this is more likely due to a recent release of LAMS than of Looker. I am currently looking into the issue, but in the meantime, can you pin your LAMS version to v1 rather than `latest` (v2) and let me know if this resolves your issue?

**Comment by fabio-looker** [comment]
@hgitta  Also, if you would be able to forward me a sample lookml file that has this issue, I could review it specifically in your case. If you're able to share it, you can email it to me at fabble@google.com

**Comment by hgitta** [comment]
Thanks for the response ; I apologies for delay in response, emailed you a copy of sample file .

**Comment by fabio-looker** [comment]
Thanks for bearing with me.

- Both the runtime error and reported parsing issues should be fixed in [v2.0.3](https://www.npmjs.com/package/@looker/look-at-me-sideways/v/2.0.3)
- **The default output format is different from v1, so even with these bugfixes v2 is not a drop-in replacement for v1**
- Docs have been updated with [an article about the contents of the v2 release](https://looker-open-source.github.io/look-at-me-sideways/release-notes/v2)
- To prevent similar issues in the future, I have updated any installation docs that install from NPM to include an explicit reference a major version in the installation command, and added tests to ensure these docs remain in-sync with the currently published major version.

If you still experience any unexpected errors, let me know and I will re-open the ticket.
